explorer/components/global/StoryCmip6Downscaled.vue

```diff
@@ -0,0 +1,317 @@
+      dim_scenario: 1,
+      dim_year: 1,
+    },
+    coastline: true,
+  },
+  {
+    id: 'tasmin_cmip6_2000',
+    title: '2000, CNRM-CM6-1-HR',
+    source: 'rasdaman',
+    wmsLayerName: 'cmip6_downscaled_tasmin_wms',
+    style: 'ardac_tasmin',
+    legend: 'tas',
+    rasdamanConfiguration: {
+      dim_model: 1,
+      dim_scenario: 0,
+      dim_year: 0,
+    },
+    coastline: true,
+  },
+  {
+    id: 'tasmin_cmip6_2100',
+    title: '2100, CNRM-CM6-1-HR, SSP5-8.5',
+    source: 'rasdaman',
+    wmsLayerName: 'cmip6_downscaled_tasmin_wms',
+    style: 'ardac_tasmin',
+    legend: 'tas',
+    rasdamanConfiguration: {
+      dim_model: 1,
+      dim_scenario: 1,
+      dim_year: 1,
+    },
+    coastline: true,
+  },
+]
+
+const cmip6_downscaled_legend: Record<string, LegendItem[]> = {
+  tas: [
+    { color: '#6468ac', label: '&lt;-20°C' },
+    { color: '#7394c1', label: '&ge;-20°C, &lt;-15°C' },
+    { color: '#94bcd5', label: '&ge;-15°C, &lt;-10°C' },
+    { color: '#badae5', label: '&ge;-10°C, &lt;-5°C' },
+    { color: '#e0ecf1', label: '&ge;-5°C, &lt;0°C' },
+    { color: '#f4e0a9', label: '&ge;0°C, &lt;5°C' },
+    { color: '#f4bc88', label: '&ge;5°C, &lt;10°C' },
+    { color: '#ef9073', label: '&ge;10°C, &lt;15°C' },
+    { color: '#dc6961', label: '&ge;15°C, &lt;20°C' },
+    { color: '#ba505e', label: '&ge;20°C' },
+  ],
+}
+
+const cmip6_downscaled_mapId = 'cmip6_downscaled_tas'
+mapStore.setLegendItems(cmip6_downscaled_mapId, cmip6_downscaled_legend)
+</script>
+
+<template>
+  <section class="section">
+    <div class="content is-size-5">
+      <h3 class="title is-3">
+        CMIP6: Increasing resolution through downscaling
+      </h3>
+
+      <p>
+        Raw outputs from General Circulation Models (GCMs) are helpful to get a
+        glimpse into how the climate is likely to change at a global scale, but
+        they have limitations. When zooming in on a particular region like
+        Alaska, for example, the data is very coarse (low resolution) and
+        difficult to relate to local communities and areas. This is where
+        downscaling helps.
+      </p>
+
+      <p>
+        Downscaling is the process of producing a higher-resolution dataset from
+        low-resolution GCM data. Downscaling has two flavors:
+      </p>
+
+      <ul>
+        <li>
+          <strong>Statistical downscaling:</strong> Use the historical period of
+          a high-resolution dataset that is based on observed data, compared to
+          the historical baseline of a low-resolution GCM dataset, to draw
+          inferences about how conditions are likely to change at a local scale
+        </li>
+        <li>
+          <strong>Dynamical downscaling</strong>: Use low-resolution GCM data as
+          inputs into a weather forecasting model to simulate conditions at a
+          local scale using physical laws
+        </li>
+      </ul>
+
+      <p>
+        <a href="https://uaf-snap.org/how-do-we-do-it/downscaling/"
```

Comment:
Including this link could be a little misleading because because the statistical portion of that content refers to the "Delta" method from the CMIP5 generation of products, and we've significantly departed from that. Moreover, there are specific references to spatial resolution (2 km) and even older CMIPs (scenario A1B) that might confuse the issue.